### PR TITLE
Customized button and input

### PR DESCRIPTION
**Pull request by JulietShaban**

### CODE REVIEW

File: client/src/common/themes/lightPalette.ts [hunk 1]
```diff
@@ -2,7 +2,6 @@ import {createTheme} from '@mui/material/styles';
 
 const lightPalette = createTheme({
     palette: {
-        mode: 'light',
```
Comment (mykhalenych) [inline client/src/common/themes/lightPalette.ts:5]: why did you delete it?

File: client/src/components/common/Button/Button.tsx [hunk 2]
```diff
@@ -0,0 +1,22 @@
+import React from 'react';
+import type {ButtonProps} from '@mui/material';
+
+import StyledButton from './styles';
+import {CustomButtonProps} from './types';
+
+const Button: React.FC<CustomButtonProps & ButtonProps> = ({
+    minWidth = 240,
+
```
Comment (mykhalenych) [inline client/src/components/common/Button/Button.tsx:9]:
```suggestion
```

**Review by mykhalenych — CHANGES_REQUESTED** [review]

File: client/src/common/themes/lightPalette.ts (hunk 1 above)
Comment (JulietShaban) [reply client/src/common/themes/lightPalette.ts:5]: it was deleted accidentally

**Review by mykhalenych — APPROVED** [review]
